cli/command/container/formatter_stats.go

```diff
@@ -13,11 +13,13 @@ const (
 	winOSType                  = "windows"
 	defaultStatsTableFormat    = "table {{.ID}}\t{{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.MemPerc}}\t{{.NetIO}}\t{{.BlockIO}}\t{{.PIDs}}"
 	winDefaultStatsTableFormat = "table {{.ID}}\t{{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.NetIO}}\t{{.BlockIO}}"
+	autoRangeStatsTableFormat  = "table {{.ID}}  {{.MemUsage}}  {{.AutoRange}}"
 
 	containerHeader = "CONTAINER"
 	cpuPercHeader   = "CPU %"
 	netIOHeader     = "NET I/O"
 	blockIOHeader   = "BLOCK I/O"
+	autoRangeHeader = "  Cur MIN/MAX      Opti SOFT/HARD     | CPUS /  %  /  Time"
```

Comment:
I'm not familiar with the container stats formatter, but it feels weird having these spaces at the beginning and all these lower cases letters (all other headers are fully upper case).
It also feels like it is 3 different columns( `Cur MIN/MAX`, `Opti SOFT/HARD` and `CPU / % / Time`)? Shouldn't they be 3 different headers? And what is this `|` for before CPUS? (maybe I'm missing something 😅 )

Reply:
You are right, should have wrote everything in caps and yes they should, it totally make sense.
The logic behind this was:
I didn't wanted to modify the formatter too heavily implementing new vars that weren't useful for every users, so i did it kinda custom as you see. But the spacing is kinda bad so i should maybe do it differently. Implementing a var for each Headers is a good start, as it should help keep the spacing right